swagger_spec_validator/validator20.py

```diff
@@ -35,10 +36,14 @@ def validate_spec(spec_json, _spec_url=None):
     """
     validate_json(spec_json, 'schemas/v2.0/schema.json')
 
+    # Dereference all $refs so we don't have to deal with them
+    fix_malformed_model_refs(spec_json)
```

Comment:
I do not think this is the right place to add it. If `ssv` is been used through [`bravado`](https://github.com/Yelp/bravado-core/blob/49a322f5aec84f7c50409c136f82888ed96b26f0/bravado_core/spec.py#L87,L91) and validations are ON, this flow gets duplicated. I think dereferencing should be a util function somewhere that can be called from both `ssv` and `bravado`, and `validate_spec` when called from bravado can safely assume it has already been derefenced by now. Thoughts?